service/src/test/java/bio/terra/pipelines/controller/PipelinesApiControllerTest.java

```diff
@@ -116,7 +116,7 @@ void getPipelineDetailsOkNoVersion() throws Exception {
     assertEquals(TestUtils.TEST_PIPELINE_1.getPipelineType(), response.getType());
     assertEquals(TestUtils.TEST_PIPELINE_1.getVersion(), response.getPipelineVersion());
 
-    // check that the response only includes user-provided inputs
+    // check that the response includes user-provided inputs, and outputs
```

Comment:
the "only" here is to ensure all inputs that are in the response are `isUserProvided == true`.  Might be nice to fit that back in here english wise 😄